src/app/lib/layoutGrid.js

```diff
@@ -162,3 +182,78 @@ export const layoutGridItemSmall = css`
 export const layoutGridItem = css`
   grid-column: 1 / -1;
 `;
+
+// if the specified grid span is wider than the maximum width the grid will expand
+// with an extra grid-gap, so we need this to prevent that from happening
+const specifiedOrMaximum = (specified, maximum) =>
+  specified > maximum ? maximum : specified;
+
+export const nestedGridItemSmallCss = css`
```

Comment:
It would be good to add a brief section in the README.md file in this directory about how the nesting works. 
Specifically about the usage of `<NestedGridItemSmall>`, `<NestedGridItemMedium>`, `<NestedGridItemLarge>`. e.g. stating that `NestedGridItemSmall` should only be used within `ConstrainedGridItemSmall` etc.